charts/pvault-server/templates/deployment.yaml

```diff
@@ -32,8 +32,7 @@ spec:
       labels: {{- include "pvault-server.labels.standard" (dict "customLabels" (include "pvault-server.pod.labels" .) "context" .) | nindent 8 }}
     spec:
       {{- with .Values.imagePullSecrets }}
-      imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
+      imagePullSecrets: {{- toYaml . | nindent 8 }}
```

Comment:
Does that still need the nindent?

Reply:
Yes, it is still a dict, put it inline just to align with the `{{- include` pattern.